src/View/mainpage/DrawROIWindow.py

```diff
@@ -548,10 +541,7 @@ def update_metadata(self):
             slice_pos = imagePosPatientCoordinates[2]
 
         # For formatting
-        if self.window.zoom == 1:
-            zoom = 1
-        else:
-            zoom = float("{0:.2f}".format(self.window.zoom))
+        zoom = 1
```

Comment:
deliberate elimination of support for window zoom?
That's OK, I'd have to look at the commit comments to know why.